apollo-portal/src/main/resources/static/scripts/directive/namespace-panel-directive.js

```diff
@@ -352,6 +352,7 @@ function directive($window, $translate, toastr, AppUtil, EventManager, Permissio
                                     publicNamespace.hasPublishedItem = true;
                                 }
                             });
+                            publicNamespace.isPropertiesFormat = publicNamespace.format == 'properties'
```

Comment:
I suggest appending a `;` to be consistent with other codes.
```suggestion
                            publicNamespace.isPropertiesFormat = publicNamespace.format == 'properties';
```